src/microbe_stage/MicrobeStage.cs

```diff
@@ -1050,6 +1051,9 @@ private void MakeEditorForFreebuildAvailable()
         if (PlayerIsEngulfed(Player))
             return;
 
+        if (!CurrentGame!.FreeBuild)
```

Comment:
Did I forget to actually check the game was in freebuild... very good catch. I suppose it'll be easier to have this check here than in the two places I called this as that might bet a bit hard to follow the logic there.